apps/dotcom/sync-worker/src/UserDataSyncer.ts

```diff
@@ -186,6 +186,12 @@ export class UserDataSyncer {
 
 	private updateStateAfterBootStep() {
 		assert(this.state.type === 'connecting', 'state should be connecting')
+		if (this.state.data) {
```

Comment:
We should remove the broadcast inside the if statement below so that we don't send twice.

Reply:
if there are buffered events it will need to broadcast again. we could check whether or not it's necessary though

Reply:
We still send the state data there and it doesn't look like those buffered events ever make it there?

Reply:
ah another good catch, fixed!